### PR TITLE
Remove old logo

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: common/colors.tex [hunk 1]
```diff
@@ -0,0 +1,3 @@
+\definecolor{tu}{RGB}{132,182,24}
```
Comment (The-Ludwig) [inline common/colors.tex:1]:
```suggestion
\definecolor{tu}{RGB}{132,182,24}
\definecolor{tu-alt}{RGB}{99,154,0}
```
And I would even argue to define `tu` and `tu-alt` the other way around. The cooperate design specifies, that this is the TU-green and the other is for "Microsoft" products. We should follow the cooperate design guide, not what we find preferable. https://itmc.tu-dortmund.de/unsere-services/medien-services/grafik/design/corporate-design/

**Review by The-Ludwig — APPROVED** [review]

File: common/colors.tex (hunk 1 above)
Comment (maxnoe) [reply common/colors.tex:1]: There are two colors on the current TU websites, the second one here is used for buttons and large areas.

According to the TU corporate design document here:
https://service.tu-dortmund.de/documents/18/1022595/130415_corporate_design_manual_TUDortmund.pdf/539f1b75-806a-44fa-8881-503260233ddc

The main color is 

> Pantone 368 C
> CMYK 57/0/100/0
> (RAL 6018)
> HEX #83B818

Converting that hex code to 8-bit integers is 132, 182, 24 and it matches the color inside the TU logo.

I'd use the "logo color".

I can write  a mail to the marketing, but I just think there is a mistake on their site actually...

Comment (The-Ludwig) [reply common/colors.tex:1]: No no Email, it's ok. I found the pitfall, error on my side: 

>Die Farben im Web unterscheiden sich geringfügig von
den Farben im Print-Corporate-Design. Die Anpassung
der Farbwerte im Web dient der Barrierefreiheit des
Internetauftritts.

Comment (The-Ludwig) [reply common/colors.tex:1]: Thanks for the link

Comment (The-Ludwig) [reply common/colors.tex:1]: Question is: Is this print or is this web? (Nobody will print it, everybody will view it on their screen) Anyway enough of this, I am fine with it. We can define `tu-alt` as `tu-web` so people keep in mind, that there is a seperate web-color...

Comment (maxnoe) [reply common/colors.tex:1]: > Die Anpassung
der Farbwerte im Web dient der Barrierefreiheit des
Internetauftritts.

Warum ist ein leicht anderes Grün Barrierefreier???

Also, wenn der Grund rot-grün schwäche ist, warum ändert man die Hauptfarbe des CDs und nicht die anderen....? egal.

Wir haben eine Akzentfarbe, ich finde da sollten wir die "richtige" nehmen

**Pull request merged**